Gemfile

```diff
@@ -1,5 +1,7 @@
+# frozen_string_literal: true
+
 source 'http://rubygems.org'
 
 gemspec
 
-gem 'rubocop', '0.67.2'
+gem 'rubocop'
```

Comment:
I wonder if not specifying a version will lead to drift between CI and local development in the future, given the lockfiles are not checked in. I.e. when the next version of rubocop is released,

- CI is in a clean state and `bundle install` will install the latest version.
- Locally devs might already have an older version installed and `bundle install` would not update rubocop to the latest.

Reply:
Yeah this is a good point - how do you think we should approach this? Should I go back to pinning to the most recent version?

Reply:
Yeah I'm not sure about the best practice, and it seems like some popular gems are at least specifying some versions (e.g. [thor](https://github.com/rails/thor/blob/efe79b599eb401d36265ed4a8b48fb60f42a06a4/Gemfile#L15), [aws-sdk-ruby](https://github.com/aws/aws-sdk-ruby/blob/0f40509d517af9c923b364547ac1bcbaa35862fa/Gemfile#L54)). Maybe we can go back and pin the latest version?